### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by ekmst**
Hello!

* Type: code quality | documentation
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change: Minor improvements

Thanks

### CODE REVIEW

**Review by sergeyklay — DISMISSED** [review]

File: phalcon/Storage/Adapter/Stream.zep [hunk 1]
```diff
@@ -167,8 +167,9 @@ class Stream extends AbstractAdapter
     }
 
     /**
-     * Returns the already connected adapter or connects to the Memcached
-     * server(s)
+     * Always returns null
+     *
+     * @return null
```
Comment (Jeckerson) [inline phalcon/Storage/Adapter/Stream.zep]:
```suggestion
     * @return string
```

Comment (sergeyklay) [reply phalcon/Storage/Adapter/Stream.zep]: @Jeckerson  Not really. It always NULL for this class

**Comment by sergeyklay** [comment]
@ekmst Could you please remove redundant import declaration from `phalcon/Storage/Adapter/Stream.zep` file:

```
use Phalcon\Storage\Adapter\AbstractAdapter;
```

**Comment by ekmst** [comment]
> AbstractAdapter

Stream inherits from AbstractAdapter

https://github.com/phalcon/cphalcon/blob/5d0c288c31fbbd7fe0ac51300579a16d38bf5aaf/phalcon/Storage/Adapter/Stream.zep#L27

File: phalcon/Filter/FilterFactory.zep [hunk 2]
```diff
@@ -11,15 +11,14 @@
...
-use Phalcon\Filter\FilterInterface;
 
 class FilterFactory
 {
     /**
      * Returns a Locator object with all the helpers defined in anonynous
      * functions
      */
-    public function newInstance() -> <FilterInterface>
+    public function newInstance() -> <Filter>
```
Comment (niden) [inline phalcon/Filter/FilterFactory.zep]: This should return a filter interface because you can write filters yourself and register them in the factory

**Review by niden — CHANGES_REQUESTED** [review]

**Comment by sergeyklay** [comment]
@ekmst you don't need import it because the same namespace

File: phalcon/Storage/Adapter/Apcu.zep [hunk 3]
```diff
@@ -111,10 +111,9 @@ class Apcu extends AbstractAdapter
     }
 
     /**
-     * Returns the already connected adapter or connects to the Memcached
-     * server(s)
+     * Always returns null
      *
-     * @return mixed
+     * @return null
```
[the remark this answers is not in the document]
Comment (sergeyklay) [reply phalcon/Storage/Adapter/Apcu.zep]: I think we can use annotation as NULL and keep return type hint as `-> var` (as a far interface declares it).

**Review by niden — DISMISSED** [review]

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @ekmst

**Pull request merged**